Sources/StreamChatUI/Utils/ChatChannelNamer_Tests.swift

```diff
@@ -0,0 +1,131 @@
+//
+//  ChatChannelNamer_Tests.swift
+//  StreamChatUI
+//
+//  Created by Dominik Bucher on 01.03.2021.
+//  Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import StreamChatTestTools
+@testable import StreamChatUI
+import XCTest
+
+class ChatChannelNamer_Tests: XCTestCase {
+
+    var defaultMembers: Set<_ChatChannelMember<NoExtraData>>! = [
+        .mock(
+            id: .unique,
+            name: "Darth Vader",
+            imageURL: nil,
+            isOnline: true
+        ),
+        .mock(
+            id: .unique,
+            name: "Darth Maul",
+            imageURL: nil,
+            isOnline: true
+        ),
+        .mock(
+            id: .unique,
+            name: "Kylo Ren",
+            imageURL: nil,
+            isOnline: true
+        )
+    ]
```

Comment:
Please get used to the idea of generating these things in `setUp()` (see more info here https://qualitycoding.org/xctestcase-teardown/ ). In this case, it's technically OK, but let's stick to a single pattern that works for all cases :)

Reply:
Fixed :)